code/game/machinery/_machinery.dm

```diff
@@ -96,44 +96,51 @@
 	layer = BELOW_OBJ_LAYER //keeps shit coming out of the machine from ending up underneath it.
 	flags_ricochet = RICOCHET_HARD
 	receive_ricochet_chance_mod = 0.3
-
 	anchored = TRUE
 	interaction_flags_atom = INTERACT_ATOM_ATTACK_HAND | INTERACT_ATOM_UI_INTERACT
 	blocks_emissive = EMISSIVE_BLOCK_GENERIC
 	initial_language_holder = /datum/language_holder/speaking_machine
+	armor_type = /datum/armor/obj_machinery
 
+	///NO_POWER,BROKEN,MAINT
```

Comment:
```suggestion
	///can be NO_POWER, BROKEN or MAINT
```

Reply:
also, if we're documenting this what is it? like what do these defines mean in the context of machines. the machine's status?

Reply:
Yes the machines status like does it have power, is it physically broken or is its panel open for maintainence.